test/recipe_box_web/controllers/cuisine_controller_test.exs

```diff
@@ -1,9 +1,37 @@
 defmodule RecipeBoxWeb.CuisineControllerTest do
   use RecipeBoxWeb.ConnCase
+  
+  alias RecipeBox.Cuisine
 
   test "GET /new", %{conn: conn} do
-    conn = get(conn, "/cuisines//new")
+    conn = get(conn, "/cuisines/new")
     assert html_response(conn, 200) =~ "Create Cuisine"
   end
+
+  test "POST /cuisines creates a new cuisine" do
```

Comment:
Does this POST request always succeed, or should there be an error test case as well?